.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "trailingComma": "es5"
```

Comment:
I don't have strong preference on prettier config. It's just this options creates fewer diffs.